src/app/controllers/validationSchemas/User.ts

```diff
@@ -13,3 +13,8 @@ export const createSchema = Joi.object({
 export const updatedSourcesSchema = Joi.object({
   sourcesOfInterest: Joi.array().items(Joi.string()).required(),
 });
+
+export const updatePasswordSchema = Joi.object({
```

Comment:
password: Joi.string().min(3).max(15).required().label('Password'),
password_confirmation: Joi.any().equal(Joi.ref('password'))
    .required()
    .label('Confirm password')
    .messages({ 'any.only': '{{#label}} does not match' })